### ISSUE TITLE
Extension Names in Extension Browser Should Display an  ... For Long Names

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/vscode#24240]
![screen shot 2017-04-07 at 9 46 25 am](https://cloud.githubusercontent.com/assets/12821956/24810413/391f0036-1b77-11e7-97fb-535c9c1f2b73.png)

1. Search for `angular`  in the extension explorer

**Bug**
Note that extension names that are too long have a hard cutoff

**Suggested Fix**
Add a `...` overflow to the name, similar to the extension description

**Comment by rishii7** [comment]
@mjbvz I would like to contribute here. As the names of the extensions are under a `flex` box we should force the `...` on its `children`, as `text-overflow` applies to `block` elements. 
```css 
.extensions-viewlet>.extensions .extension>.details>.header-container>.header {
min-width: 0;
} 
.....
.extensions-viewlet>.extensions .extension>.details>.header-container>.header>.name{
text-overflow: ellipsis;
white-space: nowrap;
overflow: hidden;
}
```
fixes the issue.
Can I submit a PR here?

**Comment by mjbvz** [comment]
@rishii7 Sure, a PR would be great. You may want to take a look at #24242 for this as well since the problems are related to this one

I believe @joaomoreno or @sandy081 should be able to review any PRs here and provide help on this issue

**Comment by rishii7** [comment]
#24371

**Comment by joaomoreno** [comment]
Fixed by #24371, thanks @rishii7 !
